wagtailio/project_styleguide/templates/patterns/pages/content_page/content_page.html

```diff
@@ -0,0 +1 @@
+{% extends "patterns/base_page.html" %}
```

Comment:
This is a placeholder page to be able to view the menu in the pattern library. I imagine it will need to be updated once the BE work is done.

Reply:
Suggestion: Can we create `primary_navigation.html` in the `styleguide` directory? It would also help with review as we can set an arbitrary height as it's cut off on the pattern lib.

Reply:
Do you mean the Get Started menu is cut off?

Reply:
Chatted on Slack - i've moved this to a `header.html` template in the style guide and removed the content page template